src/test/java/com/hotels/mutantswarm/mutate/LexerMutatorStoreTest.java

```diff
@@ -133,4 +133,20 @@ public void nullStream() {
     lexerStore.getMutatorsFor(position, null);
   }
 
+  @Test
+  public void equalSame() {
+    Imposter imposter4 = new Imposter(4,"hello");
+    Imposter imposter5 = new Imposter(4,"hello");
+    boolean result = imposter4.equals(imposter5);
+    assertThat(result, is(true));
+    assertThat(imposter4.hashCode() == imposter5.hashCode(), is(true));
```

Comment:
assertThat/is etc. I'll stop commenting on this now but it applies to the entire PR.